db/migrate/20210128143531_add_primary_contact_email_to_school.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPrimaryContactEmailToSchool < ActiveRecord::Migration[6.1]
+  def change
+    add_column :schools, :primary_contact_email, :string
+  end
+end
```

Comment:
This is going to conflict with @kicferk1 's work https://github.com/DFE-Digital/early-careers-framework/pull/71/files#diff-206b706d03f65d61ef14a0064609851a14599e1a986ebffe83307e4fd219e6c2R34